docs/source/msp.rst

```diff
@@ -1,26 +1,13 @@
 Membership Service Providers (MSP)
 ==================================
 
-The document serves to provide details on the setup and best practices for MSPs.
+For a conceptual overview of the Membership Service Provider (MSP), check out
```

Comment:
Again because we have removed the system channel from this doc I think we need to refer them to the 2.2 doc if they are using the system channel/